### PR TITLE
Add a version variable in a standard way, update publishing script to account for the change, add new methods in Eva object

### LINKED ISSUE #59: Change SDK User Agent to make it more explicit/unique

### PR DESCRIPTION
**Pull request by LouisBrunner**
This PR adds a custom User-Agent to all requests (format: `Automata EvaSDK/version (language/platform)`).

Because we include the version in the header, we also need to be able to access it outside `setup.py`. I used the standard `__version__` notation and moved the `sed` replacement to another file. The version will default to `0.0.dev0` when doing dev builds.

I also added the 2 new functions added by Isaac in #58 and updated the headers.

This closes #59.

### CODE REVIEW

File: tests/Eva_api_test.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+from evasdk import Eva
+
+
+class TestEva_API:
+    def test_versions(self, requests_mock):
+        eva = Eva("example.com", "NONE")
+
+        requests_mock.register_uri(
```
Comment (Charlesworth) [inline tests/Eva_api_test.py:8]: We're currently not mocking any requests in our tests, instead requiring a back end be available to test against. Might be worth a chat as my worry is if we change the endpoint but forget to change the test, the SDK tests will pass but the SDK will not work with the new endpoint.

**Review by Charlesworth — COMMENTED** [review]
Code looks and works fine, a comment on using request mocking, I think it's worth a chat (especially as I want to write a load more tests!) to get a consistent test approach, lets chat offline

File: tests/Eva_api_test.py (hunk 1 above)
Comment (LouisBrunner) [reply tests/Eva_api_test.py:8]: I guess I should rename the tests, but the goal is to check for the User-Agent so testing against a back-end wouldn't help.

**Review by Charlesworth — APPROVED** [review]

File: tests/Eva_api_test.py (hunk 1 above)
Comment (Charlesworth) [reply tests/Eva_api_test.py:8]: Ahhhh, sorry missed that, thought you were randomly testing the version endpoint!

**Review by Charlesworth — APPROVED** [review]

**Pull request merged**